requirements.txt

```diff
@@ -36,6 +36,9 @@ filetype==1.2.0
 # for fuzzy search of assets
 polyleven==0.8
 
+# We used to only use this for packaging, but now use it for the version comparison functionality
+packaging==23.1
```

Comment:
Moved packaging as a requirement to the app itself so we can use version comparison.